resources/fcontext.rb

```diff
@@ -5,3 +5,4 @@
 default_action :addormodify
 attribute :file_spec, :kind_of => String, :name_attribute => true
 attribute :secontext, :kind_of => String
+attribute :filetype, :kind_of => String
```

Comment:
thankyou for readme. Please add validation so it'll match one of the allowed values:

I took the values from here, ignoring "a" because I'm assuming it means "all". Please LMK if it's true in RHEL and/or Ubuntu. We'll default to an empty value meaning all
http://manpages.ubuntu.com/manpages/trusty/man8/semanage-fcontext.8.html

See this for a `equal_to` exmaple: http://dougireton.com/blog/2012/12/31/creating-an-lwrp/

Also update the readme for allowed values, and either a link for the explanation or a short table e.g.
```
d: directories
l: symbolic links
```
Thanks